docs/docsite/rst/scenario_guides/guide_kubernetes.rst

```diff
@@ -9,34 +9,45 @@ Requirements
 To use the modules, you'll need the following:
 
 - Run Ansible from source. For assistance, view :ref:`from_source`.
-- `OpenShift Rest Client <https://github.com/openshift/openshift-restclient-python>`_ installed on the host that will execute the modules
+- `OpenShift Rest Client <https://github.com/openshift/openshift-restclient-python>`_ installed on the host that will execute the modules.
 
 
 Installation and use
 --------------------
 
-The individual modules, as of this writing, are not part of the Ansible repository, but they can be accessed by installing the role, `ansible.kubernetes-modules <https://galaxy.ansible.com/ansible/kubernetes-modules/>`_, and including it in a playbook.
+The Kubernetes modules are part of the `Ansible Kubernetes collection <https://github.com/ansible-collections/community.kubernetes>`_.
 
-To install, run the following:
+To install collection, run the following:
 
 .. code-block:: bash
 
-    $ ansible-galaxy install ansible.kubernetes-modules
+    $ ansible-galaxy collection install community.kubernetes
 
 Next, include it in a playbook, as follows:
 
 .. code-block:: bash
 
     ---
     - hosts: localhost
-      remote_user: root
-      roles:
-        - role: ansible.kubernetes-modules
-        - role: hello-world
+      collection:
+      - community.kubernetes
```

Comment:
we've been preferring FQCN to using the collection keyword. What are your thoughts on that?